website/frontend/templates/docs/plugin-api.html

```diff
@@ -71,6 +75,31 @@ <h1 id="file_formats">File Formats</h1>
 
       register_format(MyFile)</pre>
 
+	  <p>
+		Variables explanation:
+		<li>"PLUGIN_DESCRIPTION" should be as simple as possible, while still describing the main function</li> 
+		<li>"PLUGIN_VERSION" should be filled with the version of Plugin</li>
+		<li>"PLUGIN_API_VERSIONS" should be set to the version of Picard this plugin to run with.</li>
+		<li>"PLUGIN_LICENSE" should be set with the license name of the plugin</li> 
+		<li>"PLUGIN_LICENSE_URL" should be set with the license url</li>
+		<br>
+		<br>
+		PLUGIN_API_VERSIONS available:
+		<ul>v1.0
+		<li>0.15.0</li>
+		<li>0.15.1</li>
+		<li>0.16.0</li>
+		<li>1.0.0</li>
+		<li>1.1.0</li>
+		<li>1.2.0</li>
+		<li>1.3.0</li>
+		<li>1.4.0</li>
+		</ul>
+		<ul>v2.0
+		<li>2.0</li>
+		</ul>
+	  </p>
+	  <br>
```

Comment:
No need for the `<br>` here. Preceding HTML entity is a `<p>` block and the following is a `<h1>`, which should ensure that there's some whitespace between the two.